provider/google-docs/index.ts

```diff
@@ -38,7 +38,7 @@ const googleDocs: Provider<Settings> = {
         const docsAPI = googleDocsAPI({ version: 'v1', auth: oauth2Client })
         const driveAPI = googleDriveAPI({ version: 'v3', auth: oauth2Client })
 
-        const quotedQuery = JSON.stringify(params.query)
+        const quotedQuery = JSON.stringify(params.message)
```

Comment:
As a TODO we need to update this provider to match the new API. If I am not mistaken this probably works better when run as a mention.

Also reading this provider makes me think message may be useful as part of mention as well. eg the search query below could do an or between the query or the chat message.